pkg/kapis/devops/v1alpha3/handler.go

```diff
@@ -189,10 +189,18 @@ func (h *devopsHandler) GetPipeline(request *restful.Request, response *restful.
 
 func (h *devopsHandler) ListPipeline(request *restful.Request, response *restful.Response) {
 	devopsProject := request.PathParameter("devops")
+	creator := request.QueryParameter("creator")
```

Comment:
What should we do if we want to support query the Pipeline list by createTime, status, and more?

Reply:
Thanks for your reminder. I'm working on this flexible scenario.